distribution/docs/src/main/jdocs/content/_plugins/post-process-plugin-intro-contents.adoc

```diff
@@ -0,0 +1,17 @@
+:type: pluginIntro
+:status: published
+:title: Post-Process Plugins
+:link: _post_process_plugins
+:summary: Performs additional processing after ingest.
+:plugintypes: postprocess
+:order: 06
+
+[NOTE]
+====
+This code is experimental. While this interface is functional and tested, it may change or be removed in a future version of the library.
```

Comment:
I'm fine with simplifying the note, but I think it should be generalized so that the same warning appears on documentation for features in "experimental" status. This will create an association between `experimental` and `subject to change`. Tangentially, it will make the warning easier to find when a component "graduates" and we need to go back in and remove the designation.